src/features/vote/RequestsWrapper.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsxImportSource @emotion/react */
+import tw, { styled } from "twin.macro"; // eslint-disable-line
+
+// Past and Upcoming have the same wrapper styles.
+const RequestsWrapper = styled.div`
+  font-family: "Halyard Display";
+  background-color: #fff;
+  ${tw`max-w-7xl mx-auto py-5 my-10 mb-10`};
+  .header-row {
+    max-width: 1350px;
+    margin: 0 auto;
+    padding-bottom: 2rem;
+    ${tw`flex items-stretch p-10`}
+    > div {
+      ${tw`flex-grow`}
+    }
+    .big-title {
+      font-size: 2.5rem;
+      font-weight: 600;
+      line-height: 1.38;
+      background-color: white;
+    }
+  }
+  .table {
```

Comment:
I suggest we make each of those a styled component instead of class selector, some of those have a low specificity and could be targeted accidentally up the tree 
![image](https://user-images.githubusercontent.com/29527327/114874227-682c8680-9dfc-11eb-8d3c-3bcf516a2cea.png)
Above there is an example where it breaks, it is a bit of a nitpick, but I could imagine a more elaborate real life example where some properties are inherited and some aren't

Reply:
Unfortunately, making a dozen components or something are just divs with style css seems a bit of a confusing mess. Do you have a suggestion?  Because the Past + Upcoming requests share a lot of the same css. The standard solution here would just be some sort of global (S)CSS classes to pull from.